src/libcadet/model/binding/AffinityComplexTitrationBinding.cpp

```diff
@@ -0,0 +1,253 @@
+// =============================================================================
+//  CADET
+//  
+//  Copyright © 2008-2022: The CADET Authors
```

Comment:
```suggestion
//  Copyright © 2008-present: The CADET Authors
```

@jbreue16 We should check our templates / guides and update accordingly.

Reply:
its fine in the guide, but apparently two other isoterms still had the old statement.. i guess it might have been copied from there